assignment1/numpy.ipynb

```diff
@@ -134,16 +134,16 @@
    },
    "outputs": [],
    "source": [
-    "# creates an vector with elements [0, 1, 2, 3, 4, 5]\n",
+    "# creates an vector with elements [0, 1, 2, 3, 4]\n",
     "a = np.arange(5)\n",
     "print(\"a = {}\".format(a))\n",
     "\n",
-    "# we can also set its starting point. So to create vector [10, 11, 12, 13, 14, 15]\n",
+    "# we can also set its starting point. So to create vector [10, 11, 12, 13, 14]\n",
     "v = np.arange(10, 15)\n",
     "print(\"v = {}\".format(v))\n",
     "\n",
     "# and also we can set step size. So to create a vector with elements spaced by 10\n",
-    "# and starts at 10 and ends at 50\n",
+    "# and starts at 10 and ends at 40\n",
```

Comment:
@peto2006 What do you think about adding the actual array to this line? This line could then read then read 

`# and starts at 10 and ends at 40, that is [10, 20, 30, 40]`

Reply:
I changed that line to
*# and starts at 10 and ends at 40, that is [10, 20, 30, 40]*